src/test/java/uk/gov/hmcts/reform/userprofileapi/util/UserProfileMapperTest.java

```diff
@@ -73,10 +80,15 @@ public void test_mapIdamUpdateStatusRequestWhenStatusIsSuspended() {
 
     @Test
     public void test_mapUpdatableFieldsForReInvite() {
-        UserProfile userProfile = new UserProfile();
-        UserProfileMapper.mapUpdatableFieldsForReInvite(userProfileCreationData, userProfile);
-        assertThat(userProfile.getFirstName()).isEqualTo(userProfileCreationData.getFirstName());
-        assertThat(userProfile.getLastName()).isEqualTo(userProfileCreationData.getLastName());
+        UserProfile userProfileMock = mock(UserProfile.class);
+        when(userProfileMock.getFirstName()).thenReturn(userProfileCreationData.getFirstName());
+        when(userProfileMock.getLastName()).thenReturn(userProfileCreationData.getLastName());
+
+        UserProfileMapper.mapUpdatableFieldsForReInvite(userProfileCreationData, userProfileMock);
+        
+        assertThat(userProfileMock.getFirstName()).isEqualTo(userProfileCreationData.getFirstName());
+        assertThat(userProfileMock.getLastName()).isEqualTo(userProfileCreationData.getLastName());
+        verify(userProfileMock, times(1)).setLastUpdated(any(LocalDateTime.class));
```

Comment:
if possible add one more verify for when condition

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-user-profile-api/pull/385/commits/04d04c3144cda8c41c2c8b875f3a62b0fa855501